### ISSUE TITLE
GEM: help pages unreadable in dark mode

### ISSUE THREAD

**Issue by dromer** [plugdata-team/plugdata#1455]
Because all these help files have very bright backgrounds and dark mode switches to a light font we cannot read them.

What would be a good solution here? should we just remove these backgrounds? I know this would be a lot of work to manually adjust though.

But it would be great if the help files are readable for any style/mode.

**Comment by 60-hz** [comment]
The only way I see right now is to use gray instead of white for dark mode comments. here is the result, we can read it, except for the fact that the comments are still a bit of the wrong size with vanilla...

<img width="704" alt="Capture d’écran 2024-02-24 à 17 47 27" src="https://github.com/plugdata-team/plugdata/assets/5449719/c81d579a-07c1-4cfb-80d5-0527011eca00">

**Comment by dromer** [comment]
Yeah better comment rendering in general would also be nice of course, still quite some divergence from pd on that.

So adjusting the `Comment text` setting to something like `#808080` is indeed a usable compromise for now. Maybe that should be the default for the dark themes? (I'm using `fangs` right now)
